### DISCUSSION TITLE
Update password by authenticated user

### DISCUSSION THREAD

**Discussion by oscar-costa** [pocketbase/pocketbase#2869] [Q&A]
Hi everyone, 
What type of API rule (Update rule) could I use, to alow a registered and authenticated user to change/update is password, and prevent it to change any other user field?

**Comment by ganigeorgiev** [comment]
You'll have to exclude every other field `@request.data.field:isset = false && ...` (_or at least I can't think of other approach at the moment_).

If you want to allow your users to be able to update **only** their password, then you can "lock" the update rule and use the "Request password reset" email flow as an alternative.

**Reply by oscar-costa** [reply]
Understood. Thank's
